### PR TITLE
docs: angular plugins list rewrite

### PR DESCRIPTION
**Pull request by sympatheticmoose**
**What is this feature?**

- Replaces verbose information with simple table
- Expands list with plugins which have migrated or have been deprecated

**Why do we need this feature?**

To better help users migrate from angular based plugins

**Who is this feature for?**

All users

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by josmperez — CHANGES_REQUESTED** [review]
Wow, what a difference. This doc is much improved in readability. In my commit, I have made edits in the introductory sections and made the following changes in the table: (1) Started descriptions with verbs for parallelism, 2) Added a few extra words such as "supported" or "the" for readability or to remove ambiguity. Although an argument can be made that the leading word in the description (such as "Migrate -") ought to be a new column in itself, this isn't a blocking issue.

File: docs/sources/developers/angular_deprecation/angular-plugins.md [hunk 1]
```diff
@@ -16,187 +16,193 @@ description: An annotated list of Grafana plugins using AngularJS.
...
+{{% admonition type="note" %}}
+If you want to add any specific migration guidance for your plugin here or update our assessment, please open a PR by clicking **Suggest an edit** at the bottom of this page.
+{{% /admonition %}}
 
 ## Private plugins
 
-Grafana OSS and Grafana Enterprise support the creation of private plugins for use on local instances. These plugins may have a dependency on Angular and require an update.
+Grafana OSS and Grafana Enterprise support the creation of private plugins for use on local instances. These plugins may have a dependency on AngularJS and require an update.
 
 The open source tooling listed above will include private plugins in its report **if the Grafana version is v10.1.0 or later**.
```
Comment (josmperez) [inline docs/sources/developers/angular_deprecation/angular-plugins.md:45]: Which open source tooling specifically?

Comment (sympatheticmoose) [reply docs/sources/developers/angular_deprecation/angular-plugins.md]: made a fix to clarify this

**Comment by sympatheticmoose** [comment]
@josmperez the only significant change I've made to your suggestions is changing:
> Note that minimum supported version

to 

> Note the minimum version for React

as its not really about support, but the migration.

**Review by tolzhabayev — APPROVED** [review]

File: docs/sources/developers/angular_deprecation/angular-plugins.md [hunk 2]
```diff
@@ -16,708 +16,193 @@ description: An annotated list of Grafana plugins using AngularJS.
 
 # Plugins using AngularJS
 
-The use of AngularJS in Grafana has been [deprecated]({{< relref "../angular_deprecation" >}}) and support for it will be removed in a future release.
+The use of AngularJS in Grafana has been [deprecated]({{< relref "../angular_deprecation" >}}) in favor of React. Support for AngularJS will be removed by default in Grafana 11.
```
Comment (tolzhabayev) [inline docs/sources/developers/angular_deprecation/angular-plugins.md:19]: removed -> turned off by default?

**Review by josmperez — APPROVED** [review]

**Pull request merged**